docs/self-hosted/kratos/configuration/oidc.mdx

```diff
@@ -0,0 +1,32 @@
+---
+id: oidc
+title: Social Sign In
+---
+
+Follow these steps to add a generic provider as a social sign-in provider when self-hosting Ory Kratos. First, you need to know
+the redirect URL:
+
+```
+http(s)://<domain-of-ory-kratos>:<public-port>/self-service/methods/oidc/callback/<social-signin-provider-id>
+```
+
+Next,
+
+1. create a client at your generic provider to get a Client ID and Client Secret;
+2. set the redirect URI to URL that follows this pattern:
+   `http(s)://<domain-of-ory-kratos>:<public-port>/self-service/methods/oidc/callback/<social-signin-provider-id>`;
+3. create a [Jsonnet code snippet](#data-mapping) to map the desired claims to the Ory Identity schema;
+4. encode the Jsonnet snippet with [Base64](https://www.base64encode.org/) or store it in a location available to your Ory Kratos
+   instance;
+5. add the [social sign-in provider configuration](#configuration-with-ory-cli) to the Ory Kratos configuration. Add the Jsonnet
+   snippet with mappings as a Base64 string or provide a path or an URL of the file.
```

Comment:
Please fix punctuation and capitalization in this list. 
List items should start with capital letters. Since these are proper sentences, each of them must end with a period. 
Don't use semicolons as the ending punctuation for list items. 

More: https://learn.microsoft.com/en-us/style-guide/scannable-content/lists#punctuation

Reply:
This should go in the style guide :)